src/Sentry.Unity.Editor/Android/AndroidManifestConfiguration.cs

```diff
@@ -175,6 +175,12 @@ internal void SetupSymbolsUpload(string unityProjectPath, string gradleProjectPa
             if (disableSymbolsUpload)
             {
                 symbolsUpload.RemoveUploadFromGradleFile();
+
+                if (_options!.Il2CppLineNumberSupportEnabled)
```

Comment:
This `!` looks dangerous. 

On line 160 you do `if (_options is null ||` then you `disableSymbolsUpload = true`
So it could be that options is null here. And the compiler knows, that's why you had to do `!`.

Reply:
Of course! Thanks for the catch!